src/main/java/com/sintern/api/ApplicationController.java

```diff
@@ -63,4 +66,14 @@ public void addApplication(@RequestBody AddApplicationRequest addApplicationRequ
 
         return applicationByInternPositionResponseList;
     }
+
+    @PostMapping("/applied")
+    @ResponseStatus(HttpStatus.OK)
+    public GetApplicationByOpenInternPositionAndStudentResponse findApplicationByOpenInternPositionIdAndStudentId(@RequestBody ApplicationByOpenInternPositionAndStudentRequest applicationByOpenInternPositionAnsStudentRequest) {
```

Comment:
I think there is a typo in applicationByOpenInternPositionAnsStudentRequest :D (_AnsStudent_)

Reply:
Ups, thank you